<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketWinClass</class>
 <widget class="QMainWindow" name="MarketWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>585</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>日志(双击复制)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_state">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_updata">
        <property name="text">
         <string>实时更新日志</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>日志类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_ok">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>成功</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="check_intoGroup">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>进群</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="check_keyWord">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>触发关键词</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_error">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>失败</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="check_quit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>退群</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="check_system">
          <property name="text">
           <string>系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_filter">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>过滤</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_clearLog">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>清空日志</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>今发:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_countDay">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>累发:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_countAll">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>群数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_countGroup">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_updateList">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="button_smtpList">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送SMTP设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_wordList">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送内容设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_groupList">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>QQ群设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_keyWordGroup">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>群消息关键词</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_other">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>其他设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_system">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>系统设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_update">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>在线更新</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="button_emailTest">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>邮箱测试</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MarketWin.qrc"/>
 </resources>
 <connections/>
</ui>
